app/views/shared/inboxes/_individual_message.html.erb

```diff
@@ -1,23 +1,56 @@
-<tr tabindex="0" onkeydown="handleButtonKeyDown(event, 'message_inbox')", id="message_inbox" class='<%=message.message_read==true ? "msg-inbox-read" : "msg-inbox-unread"%>' onclick="showMessage('<%= retrieve_show_path(provider, message) %>');">
-  <td style="font-weight:bold">
-    <%= message.try(:from) if @folder == 'Inbox' %>
-    <%= message.try(:to)   if @folder == 'Sent'%>
-  </td>
-  <td>
-    <%=message.created_at.in_time_zone("Eastern Time (US & Canada)").strftime("%m/%d/%Y") %>
+<% if @bs4 %>
+  <tr tabindex="0" onkeydown="handleButtonKeyDown(event, 'message_inbox')", id="message_inbox" class='msg-inbox <%=message.message_read==true ? "msg-inbox-read" : "msg-inbox-unread"%>' onclick="showMessage('<%= retrieve_show_path(provider, message) %>');">
```

Comment:
I think we should move the onclick into a script

Reply:
https://github.com/ideacrew/enroll/pull/4031/commits/965ebe23df3e72e386a566f00cba30c3344630f7#diff-85d9f92e5d3bed01e04096dd5fdfd404a78d94aea9a28076f504c8bd119230e0R2-R42